app/backend/approaches/chatreadretrievereadvision.py

```diff
@@ -154,10 +154,9 @@ async def run_until_final_call(
             user_content.append({"text": "\n\nSources:\n" + content, "type": "text"})
         if include_gtpV_images:
             for result in results:
-                if result.sourcepage:
-                    url = await fetch_image(self.blob_container_client, result)
-                    if url:
-                        image_list.append({"image_url": url, "type": "image_url"})
+                url = await fetch_image(self.blob_container_client, result)
```

Comment:
The "if result.sourcepage" is repeated in fetch_image, so I removed the redundant check here.